src/nemo_safe_synthesizer/pii_replacer/replacement/scope.py

```
@@ -0,0 +1,218 @@
+
+    def random_element(self, elements: Sequence[str]) -> str: ...
+
+    def date_of_birth(self, minimum_age: int = ..., maximum_age: int = ...) -> date: ...
+
+    def building_number(self) -> str: ...
+
+    def street_name(self) -> str: ...
+
+
+def stable_hash(s: str) -> int:
+    return int(hashlib.md5(s.encode("utf-8")).hexdigest()[:8], 16)
```

Ruff (0.16.1)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)